util/linux_install.sh

```diff
@@ -68,20 +68,17 @@ elif grep ID /etc/os-release | grep -q 'arch\|manjaro'; then
 		wget \
 		zip
 	git clone https://aur.archlinux.org/dfu-programmer.git /tmp/dfu-programmer
-	cd /tmp/dfu-programmer
+	cd /tmp/dfu-programmer || exit 1
 	makepkg -sic
 	rm -rf /tmp/dfu-programmer/
 
 elif grep ID /etc/os-release | grep -q gentoo; then
-	echo GENTOO_WARNING | fmt
-	echo -n "Proceed (y/N)? "
-	old_stty_cfg=$(stty -g)
-	stty raw -echo
-	answer=$( while ! head -c 1 | grep -i '[ny]' ;do true ;done )
-	stty $old_stty_cfg
-	if echo "$answer" | grep -iq "^y" ;then
+	echo "$GENTOO_WARNING" | fmt
+	printf "\nProceed (y/N)? "
+	read -r answer
+	if echo "$answer" | grep -iq "^y"; then
 		sudo touch /etc/portage/package.use/qmkfirmware
-		echo "sys-devel/gcc multilib" | sudo tee --append /etc/portage/package.use/qmkfirmware > /dev/null
+		echo "sys-devel/gcc multilib" >> /etc/portage/package.use/qmkfirmware
```

Comment:
Creating the file as root and then echoing into it as a user isn't going to work.  Also, why is there even a prompt to continue for Gentoo when none of the other distros have this?

Reply:
Good catch, I omitted the `sudo`. I'll fix it right away.

Check out the [warning message](https://github.com/vomindoraan/qmk_firmware/blob/master/util/linux_install.sh#L5) at the top. The recently added Slackware distro also has a prompt for similar reasons.

Reply:
```suggestion
		sudo echo "sys-devel/gcc multilib" >> /etc/portage/package.use/qmkfirmware
```